nerfstudio/pipelines/base_pipeline.py

```diff
@@ -220,8 +220,7 @@ def __init__(
         self._model = config.model.setup(
             scene_box=self.datamanager.train_dataset.dataparser_outputs.scene_box,
             num_train_data=len(self.datamanager.train_dataset),
-            semantics=self.datamanager.train_dataset.dataparser_outputs.semantics,
-            device=device,
+            metadata=self.datamanager.train_dataset.dataparser_outputs.metadata,
```

Comment:
Looks good for this PR. I do wish, however, that this was cleaner. I guess the better thing to do is probably pass the DataParserOutputs into all the Model classes? Not too sure at the moment.